earth_enterprise/src/fusion/tools/gee_test/README

```diff
@@ -1,58 +1,90 @@
-#
-# Copyright 2017 Google Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 
 Tool for installing Fusion and Earth Server, building tutorial databases (2d, 3d, and 3d Time Machine), and running tests against these databases.
 
 The clean is harsh so you should not use it on machines where you want to save your work.
 
 The test program does not work unless the asset root /usr/local/google/gevol_test already exists. This seems to be an issue with the clean install. If it fails, you will have to run:
 sudo /opt/google/bin/geconfigureassetroot --new --assetroot  /usr/local/google/gevol_test/assets
-# You can add source volumes:
+
+You must also add source volumes with the names "src" and "tutorial", respectively:
 /usr/local/google/gevol_test/src
 /opt/google/share/tutorials/fusion
 
 sudo /opt/google/bin/geselectassetroot --assetroot  /usr/local/google/gevol_test/assets
 
+sudo /opt/google/bin/geselectpublishroot /usr/local/google/gevol_test/published_db
+
+You can check your asset root and source volumes using:
+sudo /opt/google/bin/geconfigureassetroot --listvolumes
+
+If anything is missing, you can edit the asset root and source volumes with:
+sudo /opt/google/bin/geconfigureassetroot --editvolumes
 
 Asset root:
   /usr/local/google/gevol_test/assets
 
+Source volumes:
+  /usr/local/google/gevol_test/src (src)
+  /opt/google/share/tutorials/fusion (tutorial)
+
 Publish root:
   /usr/local/google/gevol_test/published_db
 
 
 Examples:
 
+To run the tutorial steps
+
+./run_tutorial.sh
+
+To check the status
+
+./status.sh
+
+To publish after the build is finished (status.sh should output "Succeeded" for both databases first)
```

Comment:
There are three databases (so it doesn't make sense to wait for "both" to finish).